rotkehlchen/balances/historical.py

```diff
@@ -94,68 +122,341 @@ def get_assets_amounts(
 
         May raise:
         - NotFoundError if no events exist for the assets in the specified time period.
+        - DeserializationError if there is a problem deserializing an event or trade from DB.
         """
         events, _ = self._get_events_and_currency(from_ts=from_ts, to_ts=to_ts, assets=assets)
         if len(events) == 0:
             raise NotFoundError(f'No historical data found for {assets} within {from_ts=} and {to_ts=}.')  # noqa: E501
 
-        total_amount, last_event_id = ZERO, None
+        negative_balance_event = None
```

Comment:
It's not just an event. But it's a tuple with data

```suggestion
        negative_balance_data= None
```